draft-rescorla-tls-esni.md

```diff
@@ -350,10 +371,22 @@ The client then creates a PaddedServerNameList:
 ~~~~
    struct {
        ServerNameList sni;
-       opaque zeros[ESNIKeys.padded_length - length(sni)];
+       uint8 nonce[16];
+       opaque zeros[ESNIKeys.padded_length - length(sni) - 16];
```

Comment:
Because the nonce is constant-length, I wouldn't include it in the padding calculation